nvtabular/workflow.py

```diff
@@ -103,7 +103,7 @@ def op_default_check(self, operators, default_in):
         if not type(operators) is list:
             operators = [operators]
         for op in operators:
-            if op.default_in not in default_in:
+            if op.default_in != default_in:
```

Comment:
So I have some questions about this ```op_default_check``` method:

1) If we don't pass a list of operators (like ```operators``` is a single value), it seems like this method is sort of broken. It will convert the operator to a list on line 104, perform the check and then remove the operator from that list - but since the list isn't returned this has no effect. I think maybe we should return the list here so that the callers can use that instead?

2) It doesn't seem like this line handles the 'ALL' case at all ? Should it be something like 

```suggestion
            if op.default_in != default_in and op.default_in != ALL:
```
instead ?

Reply:
Your right on both points.  I intended to make both these changes originally, but somehow lost track of my "plan" here.